ansible/templates/nssp-params-prod.json.j2

```diff
@@ -0,0 +1,13 @@
+{
+  "common": {
+    "export_dir": "/common/covidcast/receiving/nssp",
+    "log_filename": "/var/log/indicators/nssp.log",
+    "log_exceptions": false
+  },
+  "indicator": {
+    "wip_signal": true,
+    "export_start_date": "2020-02-01",
+    "static_file_dir": "./static",
+    "socrata_token": "{{ nwss_wastewater_token }}"
```

Comment:
We may want to rename `nwss_wastewater_token` if we reuse it elsewhere.

Reply:
Wait, I want to confirm that nssp uses the same cdc socrata token like with nchs and nwss correct?